src/strategy/strategy.py

```
@@ -0,0 +1,468 @@
+            # Handle profit/loss conditions
+            if 'profit' in condition.lower() or 'loss' in condition.lower():
+                return self._evaluate_profit_loss_condition(row, condition, entry_price)
+            
+            # Parse simple comparison conditions
+            return self._parse_comparison_condition(row, condition)
+            
+        except Exception as e:
+            logger.warning(f"Error evaluating condition '{original_condition}': {str(e)}")
+            return False
+    
+    def _clean_condition(self, condition: str) -> str:
```

Pylint (3.3.7)

[refactor] Too many return statements (7/6)

(R0911)